src/components/Html.js

```diff
@@ -74,6 +74,7 @@ class Html extends Component {
           <meta property="twitter:card" content="summary" />
           <meta property="twitter:site" content="labzero" />
           <meta property="twitter:image" content={`${root}/tile.png`} />
+          <meta name="theme-color" content="#FFC0CB" />
```

Comment:
Is there a reason why this is needed here and in `site.webmanifest`?

Reply:
It's redundant - it takes precedence over the color specified in `site.webmanifest` but the two are the same, so theoretically it doesn't matter. I added it in because Lighthouse recommended it.